<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpgradePage</class>
 <widget class="QWidget" name="UpgradePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>525</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="addressLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>520</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(134, 134, 134);</string>
   </property>
   <property name="text">
    <string notr="true">GOPXXXXXXXXXXXX</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>100</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(40,40,40);</string>
   </property>
   <property name="text">
    <string>Account balance</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="upgradeBtn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>395</y>
     <width>120</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{background-color:#469cfc;color:#ffffff;border:0px solid rgb(64,153,255);border-radius:3px;}
QToolButton:hover{background-color:#62a9f8;}
QToolButton:disabled{background-color:#cecece;}</string>
   </property>
   <property name="text">
    <string>Upgrade</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addressNameLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>263</y>
     <width>150</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(40,40,40);</string>
   </property>
   <property name="text">
    <string>Upgrade account name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>150</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(40,40,40);</string>
   </property>
   <property name="text">
    <string>To be upgraded address</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="accountComboBox">
   <property name="geometry">
    <rect>
     <x>177</x>
     <y>140</y>
     <width>120</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>35</y>
     <width>260</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Registered account upgrade</string>
   </property>
  </widget>
  <widget class="QLabel" name="balanceLabel">
   <property name="geometry">
    <rect>
     <x>177</x>
     <y>220</y>
     <width>221</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(90, 119, 197);</string>
   </property>
   <property name="text">
    <string notr="true">0 GOP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>150</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(40,40,40);</string>
   </property>
   <property name="text">
    <string>To be upgraded account</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>178</x>
     <y>80</y>
     <width>600</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(127, 127, 127);</string>
   </property>
   <property name="text">
    <string>After upgrading to a registered account, the account name will be registered in the block chain,
 support the use of account name directly to the transaction</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="tipLabel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>140</y>
     <width>200</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 96, 82);</string>
   </property>
   <property name="text">
    <string>Insufficient balance</string>
   </property>
  </widget>
  <widget class="QLabel" name="tipLabel2">
   <property name="geometry">
    <rect>
     <x>187</x>
     <y>296</y>
     <width>600</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(171,171,171);</string>
   </property>
   <property name="text">
    <string>Only lowercase letters and numbers are allowed, and must be written in lowercase. This account 
name will be permanently registered on the Goopal block and can not be deleted and modified</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>366</y>
     <width>371</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#282828;&quot;&gt;Need to pay &lt;/span&gt;&lt;span style=&quot; font-weight:600; color:#ff503f;&quot;&gt;0.01GOP &lt;/span&gt;&lt;span style=&quot; font-weight:600; color:#282828;&quot;&gt;for the upgrade&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="correctLabel">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>263</y>
     <width>350</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(103,182,103);</string>
   </property>
   <property name="text">
    <string>The account name is available once the settings can not be changed</string>
   </property>
  </widget>
  <widget class="QLabel" name="wrongLabel">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>263</y>
     <width>340</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 96, 82);</string>
   </property>
   <property name="text">
    <string>The account name has been used, please change another</string>
   </property>
  </widget>
  <widget class="QLabel" name="wrongLabel2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>263</y>
     <width>200</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 96, 82);</string>
   </property>
   <property name="text">
    <string>More than 63 characters</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
